### PR TITLE
FIX: TypeError: undefined is not an object (evaluating 'O.current.get…

### PR DESCRIPTION
**Pull request by marcosrdz**
…ID')

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/75yw7zep5d775de2znt2emcn98

[download apk](http://filestorage.bluewallet.io:1488/2631-1614182554.apk)

File: screen/transactions/transactionStatus.js [hunk 1]
```diff
@@ -117,11 +117,11 @@ const TransactionsStatus = () => {
   }, [tx, wallets]);
 
   useEffect(() => {
-    if (wallet) {
-      setSelectedWallet(wallet.current.getID());
+    if (wallet.current) {
+      setSelectedWallet(wallet.current?.getID());
```
Comment (Overtorment) [inline screen/transactions/transactionStatus.js:121]: deepscan legitematly bitchin.
also, im not sure, is that okay to call hook conditionally here..?

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3hk5213bjybb95g69uk50en150

[download apk](http://filestorage.bluewallet.io:1488/2631-1614218533.apk)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**